src/js/controller/handlePurchaseResultToggle.js

```diff
@@ -0,0 +1,12 @@
+import {
+  renderPurchaseResultSectionColAlign,
+  renderPurchaseResultSectionRowAlign,
+} from '../view/viewPurchaseResultSection.js';
+
+export const handlePurchaseResultToggle = ({ target }) => {
+  if (target.checked) {
+    return renderPurchaseResultSectionColAlign();
```

Comment:
마찬가지로 리턴값을 사용하지 않을 것이라면 
코드에 이 의도를 드러내면 좋을것같아요.

```suggestion
    renderPurchaseResultSectionColAlign();
    return
```

Reply:
한줄을 줄이겠다는 욕심으로 불필요한 값을 retrun했네요! 수정하도록 하겠습니다! 감사합니다!!👍👍👍